### ISSUE TITLE
MoneyMask not working as expected

### ISSUE THREAD

**Issue by rochapablo** [benhurott/react-native-masked-text#47]
```
import { MoneyMask } from 'react-native-masked-text/lib/masks'
```

```
static floatToString (value) {
    const mask = new MoneyMask()
    return mask.getValue(value, { unit: '' }, value)
}
```

```
expect(HMask.floatToString(0.01)).toBe('0,01') // OK
expect(HMask.floatToString(1.11)).toBe('1,11') // OK
expect(HMask.floatToString(1000.01)).toBe('1.000,01') // OK
```

folowing the the no OK values

```
expect(HMask.floatToString(1.10)).toBe('1,10') // "0,11"
expect(HMask.floatToString(1.90)).toBe('1,90') // "0,19"
```

```
"react-native-masked-text": "^1.6.3",
```

or am I missing something?

**Comment by benhurott** [comment]
Hi,

So bizarre O.o.

I will investigate it!

Thanks for reporting.

**Comment by benhurott** [comment]
Fixed at `1.6.4` =)
